package.json

```diff
@@ -77,6 +78,7 @@
     "spellchecker": "^3.4.4",
     "testcheck": "^1.0.0-rc.2",
     "tmp": "^0.0.33",
+    "to-arraybuffer": "^1.0.1",
```

Comment:
Can you talk about the need for this dependency?

Unlike `string-to-arraybuffer`, at least this function is has a nice performance optimization we probably wouldn't have written otherwise: https://github.com/jhiesey/to-arraybuffer/blob/6502d9850e70ba7935a7df4ad86b358fc216f9f0/index.js#L3-L27